djconnectwise/sync.py

```diff
@@ -1628,8 +1638,8 @@ def _assign_field_data(self, instance, json_data):
         # converted to a GenericForeignKey and would be handled differently.
         ticket_class = models.Ticket
         try:
-            charge_id = json_data['chargeToId']
-        except KeyError:
+            charge_id = json_data.get('chargeToId')
+        except AttributeError:
```

Comment:
This wouldn't raise an attribute error. It already has a `KeyError` handling this case, you can revert it to its previous value.